src/worker_job.cc

```diff
@@ -175,6 +186,27 @@ namespace detail {
 		if(m_future.wait_for(std::chrono::seconds(0)) == std::future_status::ready) {
 			m_buffer_mngr.unlock(pkg.cid);
 
+#if CELERITY_ACCESSOR_BOUNDARY_CHECK
+			const auto data = std::get<execution_data>(pkg.data);
+			auto tsk = m_task_mngr.get_task(data.tid);
+
+			for(size_t i = 0; i < m_oob_indices_per_accessor.size(); ++i) {
+				const id<3>& oob_min = m_oob_indices_per_accessor[i][0];
+				const id<3>& oob_max = m_oob_indices_per_accessor[i][1];
+
+				if(oob_max != id<3>{1, 1, 1}) {
```

Comment:
Now that I'm reading this code again (it's the same as the device accessor OOB check), I wonder why the sentinel value for the maximum is `[1,1,1]` instead of `[0,0,0]`. If the current (task) chunk receives an empty subrange on this buffer, an access to `[0,0,0]` would be illegal is not detected here without also checking the minima (which are initialized to `SIZE_MAX`).

Does this still work when we replace the `[1,1,1]` by `[0,0,0]` (also at the location where these are initialized)?